### PR TITLE
Fixes some bugs

### LINKED ISSUE #1991: atmos-lathe can't return resources
The only category of objects the atmos fabricator can make are atmos items, which means that you cant get it to dispense metal or glass sheets, so they're basically stuck in there forever

### LINKED ISSUE #1992: Admin AntagHUD doesn't show gangsters

### LINKED ISSUE #2086: Borgs can become cultists.
Happened one time to a borg whose dead body was converted to the cult and then revived, making him a cultist. I don't think this is intended.

### LINKED ISSUE #2087: "Flowey" and "Flowey Grin" AI statuses do not work.
image

Basically the two ones at the bottom.

Clicking on them does nothing. It does not attempt to change the AI status at all.

### LINKED ISSUE #2088: Chaplain's holy armour traitor item broken
The armour claims it can be blessed and unblessed, where in the blessed state it will send lightning to hit people who touch you, and when unblessed it acts like normal crusader armour. But even though I had it in the unblessed state when I was first disarmed and then hugged both times lightning came out. 

Please fix this cause it ruined my traitor round.

### PR DESCRIPTION
**Pull request by Carbonhell**
Fixes #2088 and also makes the armor not strippable as for the trelloed idea
Fixes #2087 
Fixes #2086 
Fixes #1992 
Fixes #1991

### CODE REVIEW

File: code/game/machinery/status_display.dm [hunk 1]
```diff
@@ -248,37 +248,7 @@
...
-				set_picture("ai_urist")
-			if("Facepalm")
-				set_picture("ai_facepalm")
-			if("Friend Computer")
-				set_picture("ai_friend")
-			if("Blue Glow")
-				set_picture("ai_sal")
-			if("Red Glow")
-				set_picture("ai_hal")
+		set_picture("ai_[emotion]")
```
Comment (JohnGinnane) [inline code/game/machinery/status_display.dm:251]: I'm not sure if the string is case sensitive, but if it is you might want to lower all the text of emotion.

Also "Problems?" will give you "ai_problems?" which I don't think is a valid picture? I think maybe change "Problems?" prompt to be "trollface" or the other way around.

And I can see some other unique options like "Very Happy" needs to become "veryhappy". Replace all spaces with blank string? Then there's "friend computer", "Dorfy", "Blue glow", and "red glow", none of these are valid unless I'm mistaken

**Comment by Pyko1** [comment]
The code looks fine, but it has some balance features in it as well.

Namely the nodrop flag on the chaplains holy armor.

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 2]
```diff
@@ -673,7 +673,7 @@ var/list/ai_list = list()
 
 	if(stat == 2)
 		return //won't work if dead
-	var/list/ai_emotions = list("Very Happy", "Happy", "Neutral", "Unsure", "Confused", "Sad", "BSOD", "Blank", "Problems?", "Awesome", "Facepalm", "Friend Computer", "Dorfy", "Blue Glow", "Red Glow","Flowey","Flowey Grin")
+	var/list/ai_emotions = list("Very Happy", "Happy", "Neutral", "Unsure", "Confused", "Sad", "BSOD", "Blank", "Trollface", "Awesome", "Facepalm", "Friend Computer", "Dorfy", "Blue Glow", "Red Glow", "Flowey", "Flowey Grin")
```
Comment (JohnGinnane) [inline code/modules/mob/living/silicon/ai/ai.dm:676]: Ah I see you've replaced "Problems?" with "TrollFace", but I suspect there'll still be issues with "Friend Computer", etc.

File: code/game/machinery/status_display.dm (hunk 1 above)
Comment (Carbonhell) [inline code/game/machinery/status_display.dm:251]: Icon states allow spaces in names. I've just set all icon states to be ai_Friend Computer, ai_Trollface, ai_Very Happy etc. So it works.

File: code/modules/mob/living/silicon/ai/ai.dm (hunk 2 above)
Comment (Carbonhell) [inline code/modules/mob/living/silicon/ai/ai.dm:676]: Shouldn't be,i edited icons names too.

**Comment by Carbonhell** [comment]
Desc updated with the bugs fixed. This also includes fully fixing antag huds, proxycheck send2irc option and the bugs listed

File: code/modules/research/designs/autolathe_designs.dm [hunk 3]
```diff
@@ -154,30 +154,6 @@
...
-	category = list("initial","Construction")
-
-/datum/design/rglass
-	name = "Reinforced glass"
-	id = "rglass"
-	build_type = AUTOLATHE
-	materials = list(MAT_METAL = 1000, MAT_GLASS = MINERAL_MATERIAL_AMOUNT)
-	build_path = /obj/item/stack/sheet/rglass
-	category = list("initial","Construction")
-
```
Comment (Crystalwarrior) [inline code/modules/research/designs/autolathe_designs.dm:180]: Any reason for removing these?

**Comment by Pyko1** [comment]
naw dawg, make it strip delay. should not be impossible to strip ti when you have him buckle cuffed and shit.

**Comment by Carbonhell** [comment]
Since the armor being unremovable could sound too op,i could make it lose the nodrop flag on mob death(even tho i'm unsure on how to but i'll find a way eventually)

File: code/game/machinery/status_display.dm (hunk 1 above)
Comment (JohnGinnane) [inline code/game/machinery/status_display.dm:251]: Ah okay, this is within the icon .dmi file?

**Comment by Pyko1** [comment]
In my opinion nothing should be unremoveable if you have the time, not to mention a traitor-ling chaplain is almost unkillable as he can get this, and bomb the crematorium at round start. Then only bombs can gib him, combine this with regen ability, and hes immune to space as well.

**Comment by Carbonhell** [comment]
Changelings have a power they can buy to get an unequippable armor which acts nearly the same other than the beams.

**Comment by Pyko1** [comment]
yeah but you can at least whack them to death once they regen in cuffs. And Chaplain spawns with access to his only weakness, the crematorium.

**Comment by Carbonhell** [comment]
Keep in mind that said armor costs 15 tcs. You could buy a guardian too. This PR makes the item on par with others of the same/similar price.

**Comment by Pyko1** [comment]
Guardian gibs you, but this makes you nearly invincible on some cases.

**Comment by Carbonhell** [comment]
do not merge till headmins give their opinion and till my commit shows up

File: code/modules/research/designs/autolathe_designs.dm (hunk 3 above)
Comment (Crystalwarrior) [inline code/modules/research/designs/autolathe_designs.dm:180]: Seriously why did you remove these from the auto lathe

**Comment by Crystalwarrior** [comment]
HEY NERD WHY DID YOU REMOVE STUFF FROM THE AUTOLATHE HUH

**Comment by Carbonhell** [comment]
Because it is now ab optional that shows on the top,no more a recipe,nerd

**Comment by Carbonhell** [comment]
Commit is in,code is okayed,just need heads to OK it

**Comment by Nebyab** [comment]
ok

**Comment by Nokington** [comment]
I dunno about it being unstrippable

**Comment by Carbonhell** [comment]
how does making the flag disappear when the wearer dies sound @Nokington

**Comment by Nokington** [comment]
sure

**Comment by Carbonhell** [comment]
Done, just needs a real fast codecheck on the last commit even through i tested it ingame,then merge

**Comment by Carbonhell** [comment]
someone codecheck this and merge,it's just 3-4 more lines

File: code/game/gamemodes/cult/cult.dm [hunk 4]
```diff
@@ -191,6 +192,7 @@
 		cult_mind.current.Paralyse(5)
 		cult += cult_mind
 		cult_mind.current.cult_add_comm()
+		cult_mind.special_role = "Cultist"
```
Comment (JohnGinnane) [inline code/game/gamemodes/cult/cult.dm:195]: I wonder if we could store the "special roles" into a list instead of having this single static type. It could mean someone could be a traitor AND something else like a vampire or rev. It's unlikely/not possible now but it might be

**Comment by JohnGinnane** [comment]
Code looks good

**Comment by Carbonhell** [comment]
Merge beg
